cmake/onnxruntime_unittests.cmake

```diff
@@ -1277,18 +1277,17 @@ if (NOT onnxruntime_ENABLE_TRAINING_TORCH_INTEROP)
         list(APPEND onnxruntime_perf_test_libs ${android_shared_libs})
       endif()
       if (CMAKE_SYSTEM_NAME MATCHES "AIX")
-        list(APPEND onnxruntime_perf_test_libs onnxruntime_graph onnxruntime_session onnxruntime_providers onnxruntime_framework onnxruntime_util onnxruntime_mlas onnxruntime_optimizer onnxruntime_flatbuffers iconv re2 gtest absl_failure_signal_handler absl_examine_stack absl_flags_parse  absl_flags_usage absl_flags_usage_internal)
-    endif()
+        list(APPEND onnxruntime_perf_test_libs onnxruntime_graph onnxruntime_session onnxruntime_providers onnxruntime_util onnxruntime_framework onnxruntime_optimizer onnxruntime_flatbuffers iconv re2 gtest absl_failure_signal_handler absl_examine_stack absl_flags_parse  absl_flags_usage absl_flags_usage_internal)
```

Comment:
will this work? we now add mlas early on in the dependencies so will that resolve symbols required in onnxruntime_framework? I guess it's working for onnxruntime_common so maybe cmake re-orders things based on dependencies it knows about.

Reply:
I will remove this. OrtDevice is included in many places, including binaries that link to a shared lib. Will set the alignment when the allocator is created or when OrtDevice is returned from EP.